examples.js

```diff
@@ -0,0 +1,107 @@
+function examples() {
+
+  setTimeout(blastOff, 86400000);
+
+
+  const locations = ["Austin", "New York", "San Francisco"];
+  locations.forEach(l => {
+    doStuff();
+    doSomeOtherStuff();
+    dispatch(l);
+  });
+
+
+  function createMenu(title, body, buttonText, cancellable) {
+    // ...
+  }
+  
+  createMenu("Foo", "Bar", "Baz", true);
```

Comment:
```suggestion
function createMenu({ title, body, buttonText, cancellable }) {...}

createMenu({
  title: "Foo",
  body: "Bar",
  buttonText: "Baz",
  cancellable: true
});
```
Creo que en este caso es mejor que reciba un objeto. Si recibe muchos parametros, el pattern _RORO_ te puede servir. 
https://www.freecodecamp.org/news/elegant-patterns-in-modern-javascript-roro-be01e7669cbd/